### PR TITLE
Grouped stickies

### PR DESCRIPTION
**Pull request by jamesgorrie**
## Type

Health

## Value
Simplifies the code around building up the sticky right rail, hopefully making it easier to simplify further. This spawned from not being able to add the poetry type very easily, and @davidpmccormick not having enjoyed working with it and @gestchild having lost hair over it.

It should also stop some DOM manipulation, upping performace.

@davidpmccormick @gestchild - does this seem simpler?

## Screenshot
Same old

### All
- [ ] Demoed to the relevant people
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### UI component only
- [x] A11y tested: `npm run test:accessibility <URL>`
- [x] Browser tested: `npm run test:browsers <URL>`
- [x] Works without JS in the client
- [x] Relevant tracking/monitoring has been considered

### CODE REVIEW

**Comment by jamesgorrie** [comment]
[It's easier to see the diff here](https://github.com/wellcometrust/wellcomecollection.org/pull/962/files?w=1)

File: server/views/components/article-body/index.njk [hunk 1]
```diff
@@ -1,62 +1,67 @@
...
+                <div class="{{ {s:3, m:3, l:3} | spacingClasses({padding: ['top', 'bottom'], margin: ['bottom']}) }}">
+                  <pre class="pre border-left border-left--smoke {{ {s:3, m:3, l:3} | spacingClasses({padding: ['left', 'right']}) }}">
+                    {{- bodyPart.value -}}
+                  </pre>
+                </div>
+              </div>
             </div>
-          </div>
+          {% else %}
+          {% else %}
```
Comment (alexwlchan) [inline server/views/components/article-body/index.njk]: Double else?

Comment (jamesgorrie) [inline server/views/components/article-body/index.njk]: https://circleci.com/gh/wellcometrust/wellcomecollection.org/2074

Comment (davidpmccormick) [inline server/views/components/article-body/index.njk]: Shows what I know!

**Pull request merged**